packages/agent/app/runner.ts

```diff
@@ -0,0 +1,46 @@
+import { Operation } from 'effection';
+import { Test } from '@bigtest/suite';
+
+import { Agent, Run, TestEvent } from '../shared/agent';
+import { TestFrame } from './test-frame';
+
+export interface Runner {
+  run(command: Run): Operation<void>;
+}
+
+export function* createRunner(agent: Agent): Operation<Runner> {
+
+  let testFrame: TestFrame = TestFrame.setup('test-frame', 'test-frame.html');
```

Comment:
If testFrame is not a class then this wouldn't retain anything, meaning we wouldn't need the runner.